kafka-ui-api/src/main/java/com/provectus/kafka/ui/config/CognitoOidcLogoutSuccessHandler.java

```diff
@@ -0,0 +1,53 @@
+package com.provectus.kafka.ui.config;
+
+import java.net.URI;
+import java.nio.charset.StandardCharsets;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.server.reactive.ServerHttpResponse;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.web.server.WebFilterExchange;
+import org.springframework.security.web.server.authentication.logout.ServerLogoutSuccessHandler;
+import org.springframework.security.web.util.UrlUtils;
+import org.springframework.web.server.WebSession;
+import org.springframework.web.util.UriComponents;
+import org.springframework.web.util.UriComponentsBuilder;
+import reactor.core.publisher.Mono;
+
+@RequiredArgsConstructor
+public class CognitoOidcLogoutSuccessHandler implements ServerLogoutSuccessHandler {
```

Comment:
It's a bad idea to have provider-dependent classes. Could we just add a configurable URL for logout?

Reply:
Not really, this one is required since cognito doesn't work like others oauth providers, this custom logic is required. We have discussed this previously within the same feature in ODD.